dashboard/app/models/course.rb

```diff
@@ -77,7 +77,8 @@ def persist_strings_and_scripts_changes(scripts, course_strings)
   end
 
   def write_serialization
-    return unless Rails.application.config.levelbuilder_mode
+    # Only save non-plc course, and only in LB mode
```

Comment:
Longterm it might be good to try to consolidate these, but in the near term, we are not doing so.